### DISCUSSION TITLE
Osu!mania Hidden and Fadeout, but no stable hidden.

### DISCUSSION THREAD

**Discussion by martzoz** [ppy/osu#22346] [Ideas] [answered]
Osu!mania Hidden and Fadein function the exact same but inverse, Whereas in osu!stable, mania hidden covers more space depending on your combo. I am a long time hidden player from stable, and the transition to lazer is hard due to this.

My suggestion is, rename Hidden to 'Fadeout' and implement the old Hidden mod, where the cover gradually rises with the combo.

Also, a toggle for receptor overlay, in stable the hit receptors are hidden underneath the cover, whereas in lazer they are visible at all times.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/22355

**Reply by martzoz** [reply]
Thank you for opening an issue, bach. :)

**Comment by GamerTuruu** [comment]
yes, someone already said this. this is my only complain to change to lazer. everything else is acceptable
